elasticjob-infra/elasticjob-infra-common/src/main/java/org/apache/shardingsphere/elasticjob/infra/env/IpUtils.java

```diff
@@ -95,14 +97,15 @@ private static NetworkInterface findNetworkInterface() {
         }
         return result;
     }
-    
+
+    // For testing.
```

Comment:
What is `for testing`?

Reply:
`for testing`  note the private method has  junit test. I remove this note.